src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use * selector (it impacts performance)
It is better to use the ``body`` or ``html`` tag.

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/ead6bfe9-77e0-40d6-a35b-f87007ac9cf3)